core/src/test/java/com/datastax/oss/driver/internal/core/cql/CqlRequestHandlerTest.java

```diff
@@ -103,32 +100,21 @@ public void should_fail_if_no_node_available() {
   }
 
   @Test
-  public void should_time_out_if_first_node_takes_too_long_to_respond() {
+  public void should_time_out_if_first_node_takes_too_long_to_respond() throws Exception {
     RequestHandlerTestHarness.Builder harnessBuilder = RequestHandlerTestHarness.builder();
     PoolBehavior node1Behavior = harnessBuilder.customBehavior(node1);
     node1Behavior.setWriteSuccess();
 
     try (RequestHandlerTestHarness harness = harnessBuilder.build()) {
 
-      CompletionStage<AsyncResultSet> resultSetFuture =
+      CqlRequestAsyncHandler requestHandler =
           new CqlRequestAsyncHandler(
-                  UNDEFINED_IDEMPOTENCE_STATEMENT,
-                  harness.getSession(),
-                  harness.getContext(),
-                  "test")
-              .handle();
+              UNDEFINED_IDEMPOTENCE_STATEMENT, harness.getSession(), harness.getContext(), "test");
+      CompletionStage<AsyncResultSet> resultSetFuture = requestHandler.handle();
```

Comment:
Any particular reason to extract the `requestHandler` variable in this test?

Reply:
Not any more. At one point in the implementation, I needed it for something, but it seems I don't anymore. I'll revert that bit.